data/npc/lib/npcsystem/modules.lua

```diff
@@ -1116,7 +1116,7 @@ if Modules == nil then
 					return false
 				end
 				if shop_itemid[cid] == ITEM_PARCEL then
-					doNpcSellItem(cid, ITEM_LABEL, shop_amount[cid], shop_subtype[cid], true, false, 1988)
+					doNpcSellItem(cid, ITEM_LABEL, shop_amount[cid], shop_subtype[cid], true, false, 23782)
```

Comment:
Can you make it `local backpackID = 23782` here as well like the in the function above?
Or just make it a "global" backpackContainerId at the top of the file once for all functions to use.

Reply:
like I did now?

Reply:
I think it looks good. Did you test that?

Reply:
Just did, and not working, need to edit a few thins in npc.lua as well, I'll sort it out and send a fix

Reply:
for some odd reason the "buy with backpack" is adding an empty container (without my modified code) and I don't know how to fix it, and seems like nobody reported it until now:

![wJJc6bA](https://user-images.githubusercontent.com/4684880/67645455-a5027600-f907-11e9-9d05-a9a972885b07.gif)

if someone is able to confirm and fix it, I'll be grateful, you can even include the content from my PR and close it, I'm not able to fix it myself

Reply:
pr #2699 fixes the issue, after it gets approved (I tested and its working), I'll make a new and clean PR with my changes, you can close this one now